core/src/test/java/io/jenkins/pluginhealth/scoring/probes/ThirdPartyRepositoryDetectionProbeTest.java

```diff
@@ -75,25 +93,29 @@ void shouldFailIfOnlyThirdPartyRepositoriesDetected() {
         assertThat(probe.apply(plugin, ctx))
             .usingRecursiveComparison()
             .comparingOnlyFields("id", "message", "status")
-            .isEqualTo(ProbeResult.failure(ThirdPartyRepositoryDetectionProbe.KEY, "Third party repositories detected in the plugin"));
+            .isEqualTo(ProbeResult.success(ThirdPartyRepositoryDetectionProbe.KEY, "The plugin has no third party repositories"));
         verify(probe).doApply(any(Plugin.class), any(ProbeContext.class));
     }
 
-    @Test
-    void shouldPassIfNoRepositoriesDetected() {
+    @ParameterizedTest
+    @MethodSource("pomPathAndProbeResults")
+    void shouldFailIfThirdPartRepositoriesDetected(Path resourceDirectory, String pluginName, String[] probeResultRequirement, Map<String, ProbeResult> details) {
```

Comment:
This test case fails at https://github.com/Jagrutiti/plugin-health-scoring/blob/5dfd1ed993514027a7ee41904df2b3d27ef4a8a2/core/src/main/java/io/jenkins/pluginhealth/scoring/probes/Probe.java#L61

I do not know which value is incorrect here.

Reply:
Please, debug the code. Put a breakpoint at the beginning of that linked method and see what is happening